linkml/generators/owlgen.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, schema: Union[str, TextIO, SchemaDefinition], ontology_uri_su
             SchemaLoader(METAMODEL_YAML_URI, base_dir=META_BASE_URI, importmap=kwargs.get('importmap', None),
                          mergeimports=self.merge_imports)
         self.metamodel.resolve()
+        self.emit_prefixes: Set[str] = set()
```

Comment:
@hsolbrig doesn't seem satisfactory having to do this in every __init__ method - move up to parent and set as a default?